### PR TITLE
Feature/wnb 107/implementation signin signup screens

### PR DESCRIPTION
**Pull request by vitali-sakalou**
Sign-in and sign-up screens

### CODE REVIEW

File: src/routes/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    width: 100%;
+    &&& input {
+      border-radius: 0;
+    }
+  }
+`;
+
+const SignInButton = styled(Button)`
+  &&& {
+    display: block; //probably need additional type of button with display:block
```
Comment (letutsenik) [inline src/routes/SignIn.js:55]: I think needn't. We can overwrite the styles as you've done in this case. Please, delete the comment

**Review by letutsenik — COMMENTED** [review]
Please, move all the styles for SignUp and SignIn components to a separate file styles.js

File: src/routes/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          </Link>
+        </Form>
+        <Description>
+          <Link to="/">Забыли пароль?</Link>
+        </Description>
+      </Container>
+      <Container>
+        <Description>Нет учетной записи?</Description>
+        <Description>
+          <Link to="/">Зарегистрироваться</Link>
```
Comment (ikushlianski) [inline src/routes/SignIn.js:125]: should be `/sign-up`?

File: src/routes/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          ))}
+          <Link to="/">
+            <SignInButton size="large" secondary>
+              Зарегистрироваться
+            </SignInButton>
+          </Link>
+        </Form>
+        <Description>Есть учетная запись?</Description>
+        <Description>
+          <Link to="/">Войти</Link>
```
Comment (ikushlianski) [inline src/routes/SignUp.js:131]: `/sign-in`

File: src/components/Navigation/styles.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ export const StyledMenu = styled.ul`
   list-style: none;
   overflow: hidden;
   transition: height 0.5s;
-  height: ${props => (props.open ? '306px' : '0')};
+  height: ${props => (props.open ? '357px' : '0')};
```
Comment (ikushlianski) [inline src/components/Navigation/styles.js:22]: I guess it should be 'auto'. We don't know the final size anyways

**Review by ikushlianski — APPROVED** [review]

File: src/components/Navigation/styles.js (hunk 4 above)
Comment (vitali-sakalou) [reply src/components/Navigation/styles.js:22]: I agree. Need to fix.

File: src/routes/SignIn.js (hunk 2 above)
Comment (vitali-sakalou) [reply src/routes/SignIn.js:125]: For sure). I will fix.

File: src/routes/SignUp.js (hunk 3 above)
Comment (vitali-sakalou) [reply src/routes/SignUp.js:131]: Got it.

File: src/components/Navigation/styles.js (hunk 4 above)
Comment (vitali-sakalou) [reply src/components/Navigation/styles.js:22]: But, If we use 'auto' browser don't know the heigth of element and there will be not smooth transition in height.

Comment (ikushlianski) [reply src/components/Navigation/styles.js:22]: Do we need this smooth transition at all :)?  we'll see after we have final designs and discuss UX within the team

Comment (vitali-sakalou) [reply src/components/Navigation/styles.js:22]: Maybe for today it is not necessary. I can set 'auto' for convenience if somebody decide to add menu item and not change height each time.

Comment (ikushlianski) [reply src/components/Navigation/styles.js:22]: Nice!

**Comment by ikushlianski** [comment]
Also, please request reviews from me and Dima directly when creating pull requests: it's convenient to see what pulls you need to review right at `https://github.com/pulls/`

**Comment by ikushlianski** [comment]
I mean this: Request review

**Comment by ikushlianski** [comment]
Btw since we all ideally should request reviews from everone, you can take a look at my pull request to our Wannabook API https://github.com/Wannabook/wannabook-api/pull/4 (there, we are creating our first User model and set up DB migrations and test data). Task is WNB-25

**Comment by vitali-sakalou** [comment]
> Btw since we all ideally should request reviews from everone, you can take a look at my pull request to our Wannabook API [Wannabook/wannabook-api#4](https://github.com/Wannabook/wannabook-api/pull/4) (there, we are creating our first User model and set up DB migrations and test data). Task is WNB-25

**Comment by vitali-sakalou** [comment]
Ok. I got it.

**Pull request closed without merging**